tls/s2n_connection.c

```diff
@@ -557,6 +565,9 @@ int s2n_connection_wipe(struct s2n_connection *conn)
     conn->client = conn->initial;
     conn->server = conn->initial;
 
+    /* Setup resizeable buffers */
+    POSIX_GUARD(s2n_stuffer_growable_alloc(&conn->post_handshake.in, 0));
```

Comment:
Just wondering - why is post_handshake.in set to growable? Won't it just be set to not growable again here? https://github.com/aws/s2n-tls/blob/c1db5cc604ccf088acd4685da8ccda9ce54a3c4b/tls/s2n_post_handshake.c#L84-L89

Reply:
Good catch, there's no reason to make it growable. That was left over from my previous approach to this.

And turns out I wasn't testing static post_handshake.in properly either. I forgot to actually set up the stuffer -_- I've fixed the test and added some assertions that they're setup properly.